templates/index.html

```diff
@@ -117,7 +117,13 @@ <h1>F.A.Q</h1>
 
   <article id="changelog">
     <h1>Changelog</h1>
-    <details open>
+    <details open> 
+      <summary><h2>v2.2</h2></summary>
```

Comment:
Should be 2.1?

Reply:
2.1 is currently a beta version from my repo with a local registry.json file, which I propagated via discord. Don't want to get the update checker to get stuck there :) It should be 2.11 though and not 2.2